cmd/node/config/config.toml

```diff
@@ -384,6 +383,9 @@
            MaxBatchSize = 100
            MaxOpenFiles = 10
 
+[ValidatorStatistics]
+    CacheRefreshIntervalInSec = 60
```

Comment:
why do you need a cache refresh? the data can be taken from the trie at the time of request. If you have another request in a said interval you do return from the cache - only at the next request which is out of the limit yo get the data from the trie again.

Reply:
the idea was not to read from the trie each time, because the validator's trie contains a few KB for each validator and some MB would be read each request.

Reply:
fixed. Update only on request refresh, not always on cacheRefreshInterval timout